charts/connect/values.yaml

```diff
@@ -70,6 +70,17 @@ connect:
   # Additional annotations to be added to the service resource
   serviceAnnotations: {}
 
+  # 1Password Operator Service Account Configuration
+  serviceAccount:
+    # The name of the 1Password Conenct Operator
```

Comment:
Seems like this comment doesn't make sense, let's rephrase it to represent what this property really does.

Could you please update it for the `.Values.operator.serviceAccount.create` as well? I see there it's the same.